admin/controllers/service-manager.js

```diff
@@ -103,17 +102,17 @@ const setUpdateTiming = async (req, res, next) => {
   }
 
   try {
-    await settings.save()
-    let settingsLog = new SettingsLog()
+    await settingDataService.createOrUpdate(settings)
 
+    let settingsLog = {}
```

Comment:
does it matter if settingslong is an empty object?

Reply:
Not at this point